### ISSUE TITLE
"Ignore queue" feels unintuitive

### ISSUE THREAD

**Issue by declension** [quodlibet/quodlibet#2912]
Since #2865 was merged (apologies I couldn't feed back more on the original PR, been pretty busy), we now have some new queue options which is great.

One of them, now named _Ignore Queue_ disables the queue (thus changing how all song selection works), but remains within the queue widget, and within _its_ configuration widget. This can lead to some confusion e.g.:

### Steps
Consider a common usage pattern (at least I use it a lot):
 * Enable _Ignore queue_
 * Come back later
 * Browse some music
 * Listen to a track
 * Select some interesting other tracks (e.g. an album)
 * Press ctrl-enter to enqueue.
 * Keep browsing to something else

### Expected
These enqueued tracks / album to play next

### Actual
The next in my current browser plays.

### Ideas
I can see benefit to some for the original use-case (_to occasionally ignore the queue without having to clear it_), but I'd suggest if we keep it we do some UX work. Some ideas:
1. Rename the option itself: _disable_ sounds stronger and thus more appropriate to me here.
2. Move it out of the prefs widget for the queue. It feels unusual to find a preference saying "disable me" alongside some fine-tuning.
3. If the queue is effectively disabled, indicate this with the UI, e.g. by greying out all the tracks, or some visual indication at the menu / queue widget level maybe. 
4. Related to above, an obvious enable / disable toggle on the queue itself

**Comment by declension** [comment]
Related note: _Queue Only_ shuffle mode, with _Ignore Queue_ checked is a little bit... _mutually assured destruction_ :thinking:

**Comment by Treferwynd** [comment]
About you related note, IMHO Queue Only shouldn't be a shuffle mode, or even a plugin, but a setting.

[I proposed a selector with 3 choices:](https://github.com/quodlibet/quodlibet/issues/2856#issuecomment-405021234)

* Standard Behaviour: the queue works as it always did
* Ignore Queue: as if the queue was empty
* Only Queue: only stuff in the queue gets played. Which would include the Queue Only plugin

But @frestr (the author of #2865) didn't particularly like the selector, so he may have other ideas!

**Comment by frestr** [comment]
> If the queue is effectively disabled, indicate this with the UI, e.g. by greying out all the tracks, or some visual indication at the menu / queue widget level maybe.

I agree that the "Ignore" label is a bit confusing. The reason it is not named disable though is that my intention was not to disable the queue, but rather inhibit the automatic switching to the queue. I thought about maybe adding a tooltip like "Don't automatically switch to the queue" or something like that, but can see how that doesn't intuitively map to "Ignore".

The way I mostly use the queue now is to have both the "Keep songs" and "Ignore" buttons checked. With this I can freely switch between the queue and the main song list without having to clear the queue (and still keep the songs in it), by manually clicking on a song in either of them. So it was my intention that the queue keeps playing even though it is being "ignored".

> But @frestr (the author of #2865) didn't particularly like the selector, so he may have other ideas!

Mostly because it would seem to prevent the aforementioned use case.

Having "Keep songs" enabled and "Ignore" disabled causes some slightly strange behavior though, especially since the last location in the queue is not remembered. So I can see the need of making the behavior selection a bit more intuitive. 

I'm also not sure how well a selector would fit in the current preferences popup, which leads to...

> Move it out of the prefs widget for the queue. It feels unusual to find a preference saying "disable me" alongside some fine-tuning.

I think this is a good idea and could perhaps be done for some of the other options too, as the queue preferences popup may become a bit cluttered, especially if we add more options. It would make changing the queue behavior a bit more effort though (for the user).

> Queue Only shuffle mode, with Ignore Queue checked is a little bit... mutually assured destruction

Hah, true. Didn't think about testing that.

As I mentioned in the original PR, the new queue behavior is quite rough around the edges, so the feedback is appreciated! It's apparent that the queue is being used in quite different ways.

**Comment by Treferwynd** [comment]
>> But @frestr (the author of #2865) didn't particularly like the selector, so he may have other ideas!

> Mostly because it would seem to prevent the aforementioned use case.

I don't think so, if you select Ignore Queue it should work exactly as it does right now, the selector is just a way to avoid conflicts between Ignore Queue and Queue Only.

I'm also in favor of moving out these options.

**Comment by declension** [comment]
Thanks @frestr
